.babelrc

```diff
@@ -1,3 +1,9 @@
 {
-  "presets": ["es2015"]
+  "presets": [
```

Comment:
The pre-commit hook doesn't cover this, but would prefer if this was formatted per Prettier rules:

```
{
  "presets": [
    [
      "env",
      {
        "targets": {
          "node": 4
        }
      }
    ]
  ]
}
```

Reply:
👌 Done